confgenerator/built-in-config-linux.yaml

```diff
@@ -29,6 +32,7 @@ metrics:
       default_pipeline:
         receivers:
         - hostmetrics
-        processors: []
```

Comment:
[Optional] I wonder if we should annotate the yaml binding for these lists of ids with `flow`, to cause it to always generate a bracketed list, i.e.:
```
  service:
    pipelines:
      default_pipeline:
        receivers: [hostmetrics]
        processors: [metrics_filter]
        exporters: [google]
```

Reply:
Dude, this is not the time for beautification like that.